Snippets/Core/Core_7/UpgradeGuides/6to7/Scheduling/ExecuteScheduledTaskUsingTaskRun.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Core7.UpgradeGuides._6to7.Scheduling
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System;
+    using System.Threading.Tasks;
+
+    class ExecuteScheduledTaskUsingTaskRun
+    {
+        async Task TaskRun(IEndpointInstance endpointInstance, ILog logger)
+        {
+            #region 6to7ExecuteScheduledTaskUsingTaskRun
+            await endpointInstance.ScheduleEvery(
+                    timeSpan: TimeSpan.FromMinutes(5),
+                    task: pipelineContext =>
+                    {
+                        Task.Run(() =>
```

Comment:
Why is `Task.Run` even relevant here? I think what you want to achieve is fire & forget a scheduled task to make it independent from the transport transaction